doc/src/manual/performance-tips.md

```diff
@@ -1394,6 +1394,116 @@ Prominent examples include [MKL.jl](https://github.com/JuliaLinearAlgebra/MKL.jl
 These are external packages, so we will not discuss them in detail here.
 Please refer to their respective documentations (especially because they have different behaviors than OpenBLAS with respect to multithreading).
 
+## Execution latency, loading and precompiling time
+
+### Reducing time to first plot etc.
+
+The first time a julia function is called it will be compiled. The [`@time`](@ref) macro family illustrates this.
+
+```
+julia> foo() = rand(2,2) * rand(2,2)
+foo (generic function with 1 method)
+
+julia> @time @eval foo();
+  0.252395 seconds (1.12 M allocations: 56.178 MiB, 2.93% gc time, 98.12% compilation time)
+
+julia> @time @eval foo();
+  0.000156 seconds (63 allocations: 2.453 KiB)
+```
+
+Note that `@time @eval ...` form is explained in the [`@time`](@ref) documentation.
+
+When developing a package, it can be helpful to cache this compilation in the package caches that are "precompiled"
+during the `Pkg` installation process or before package load. To precompile package code effectively, it's recommended
```

Comment:
```suggestion
When developing a package, you may be able to improve the experience of your users with *precompilation*,
so that when they use the package the code is already compiled. To precompile package code effectively, it's recommended
```